### ISSUE TITLE
Uncaught TypeError: Cannot read property 'buffer' of undefined

### ISSUE THREAD

**Issue by zenashopson** [foreshadow/atom-python-run#58]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.21.0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.11.6
**Thrown From**: [atom-python-run](https://github.com/foreshadow/atom-python-run) package 0.9.2

### Stack Trace

Uncaught TypeError: Cannot read property 'buffer' of undefined

```
At /Users/zenashopson/.atom/packages/atom-python-run/lib/atom-python-run.js:32

TypeError: Cannot read property 'buffer' of undefined
    at run (/packages/atom-python-run/lib/atom-python-run.js:32:41)
    at HTMLElement.runF6 (/packages/atom-python-run/lib/atom-python-run.js:18:13)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:381:36)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
  5x -1:48.4.0 Python run: run-f6 (input.hidden-input)
```

### Non-Core Packages

```
atom-python-run 0.9.2 
```

**Comment by xovertheyearsx** [comment]
How do I reproduce this? Can you point me in the right direction?

**Comment by foreshadow** [comment]
Have you save (or even create) the file before running? 
It is known that occasionally you code will run before being saved. And if you have not created the file, the buffer will be undefined... I will fix it soon.

**Comment by foreshadow** [comment]
Fixed in v0.9.6
